### PR TITLE
Test greetz.nl

### PR DESCRIPTION
**Pull request by emmaaghlamazyan94**

### CODE REVIEW

File: src/test/java/GreetzTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        cardElem.click();
+        Thread.sleep(3000);
+        WebElement amount = driver.findElement(By.name("amount"));
+        amount.click();
+        amount.clear();
+        amount.sendKeys("4");
+        Double price = Double.parseDouble(driver.findElement
+                (By.xpath("//div[@class='page-detail__price']//span[@class='price-normal']"))
+                .getText().substring(2).replace(",", "."));
+        Double priceMultiply = 4 * price;
```
Comment (chandranik90) [inline src/test/java/GreetzTest.java:65]: I have noticed that the initial price changes when the item count is being increased. It means, that a potential discount can be applied to the item. Hence, the final calculated amount will be changed as well. So, for having valid test, please gather the normal price after changing the amount.

**Review by chandranik90 — CHANGES_REQUESTED** [review]

File: src/test/java/GreetzTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class GreetzTest {
+    private WebDriver driver;
+
+    @BeforeMethod
```
[the remark this answers is not in the document]
Comment (emmaaghlamazyan94) [reply src/test/java/GreetzTest.java:11]: It was just the task 1, only added BeforeMethod, but I will combine both with corrections

File: src/test/java/GreetzTest.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+
+public class GreetzTest {
+    private WebDriver driver;
+    String email = "register@gmail.com";
```
Comment (chandranik90) [inline src/test/java/GreetzTest.java:11]: Always use access modifiers. And always use private, if there is no special reason for not doing so

File: src/test/java/GreetzTest.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        driver.findElement(By.id("login-cta")).click();
+        Thread.sleep(5000);
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test(priority = -1)
```
Comment (chandranik90) [inline src/test/java/GreetzTest.java:32]: Not sure, if negative priority is acceptable :) At least I never face it. It's better to start from 0

**Review by chandranik90 — APPROVED** [review]

**Pull request merged**